src/test/java/unit/BqDatasetControlled.java

```diff
@@ -36,7 +35,7 @@ void listDescribeReflectCreate() throws IOException {
     // `terra workspace set --id=$id --format=json`
     UFWorkspace workspace =
         TestCommand.runAndParseCommandExpectSuccess(
-            UFWorkspace.class, "workspace", "set", "--id=" + getWorkspaceId());
+            UFWorkspace.class, "workspace", "set", "--id=" + getUserFacingId());
```

Comment:
Are you 100% sure it's OK to break existing scripts that use `id` to mean `uuid`? Maybe send Nicole some flowers.

Reply:
Maybe this flag should accept either UUID or UFID?

Reply:
If we had existing users, I would've kept `id` to mean uuid and used `ufid` to mean the new ID, to prevent breakage. I agree we need a way to specify the `uuid` in certain scenarios (like a user has a spreadsheet with a row for each workspace and needs a unique immutable key to update it).